File: 011_Container_With_Most_Water/011_1.py
class Solution:
    def maxArea(self, height):
        area = 0
        for i, left in enumerate(height):
            for j, right in enumerate(height[1:]):
                if left <= right:
                    low = left
                else:
                    low = right
                area_2 = (j - i) * low
                if area_2 > area:
                    area = area_2
        return area

File: 011_Container_With_Most_Water/011_1.java
class Solution
{
    public int maxArea(int[] height)
    {
        int low, area = 0, area_2;
        for (int i = 0; i < height.length; i++)
        {
            for (int j = 1; j < height.length; j++)
            {
                if (height[i] <= height[j]) low = height[i];
                else                        low = height[j];
                area_2 = (j - i) * low;
                if (area < area_2) area = area_2;
            }
        }
        return area;
    }
}